backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -63,6 +67,22 @@ public static void start(int port) {
     // services (matched in order of creation)
     AppsProxyService.create(new SqlDatabase(false));
 
+    // add trailing /
+    get(
+        "/:schema",
+        (req, res) -> {
+          res.redirect("/" + req.params("schema") + "/");
+          return null;
```

Comment:
why null , what are you trying to signal using null

Reply:
this I believe is issue with the framework that they don't allow to return redirect as a result.

Reply:
turns out it can be done